client_test.go

```diff
@@ -907,6 +907,130 @@ func TestCreatePage(t *testing.T) {
 			},
 			expError: nil,
 		},
+		{
+			name: "database parent, successful response",
+			params: notion.CreatePageParams{
+				ParentType: notion.ParentTypePage,
```

Comment:
Can you change this table test to use `ParentType: notion.ParentTypeDatabase`? The idea is that we want to assert that when you create a _database_ page, that the `type` of DatabasePageProperty's are not included in the JSON `POST` body. Right now, the test is using a _parent_ page.

Reply:
OK! I will👍